scripts/msi-installer-builder/main.go

```diff
@@ -114,6 +116,11 @@ func normalize(preset languagePreset, c *config) (*config, error) {
 	if err != nil {
 		return c, err
 	}
+
+	if c.MSIVersion == "" {
+		c.MSIVersion = msiVersionInfo()
+	}
```

Comment:
This feels dirty and pointless, we are always setting it to `msiVersionInfo()` one way or another.

Reply:
The apparent duplication seems less dirty, to me, than the function not doing it's job based on the special knowledge that we happen to set the field in current cases.

Reply:
So error out if it's empty. This is too dirty.